### DISCUSSION TITLE
Part 3 - Extractive QA: store results with the HuggingFace library

### DISCUSSION THREAD

**Discussion by juliankklotz** [sebastian-hofstaetter/teaching#140] [Advanced IR: Exercise Q&A]
Hey everyone,

We could not find out what is meant by storing the results with the HuggingFace library, as it seems that normally only the pre-trained model is stored in the library.

Best Regards
Julian

**Comment by ldosen00** [comment]
I assume you are referring to the sentence from the assignment: Implement code  in ``src/extractive_qa.py`` to load the model, tokenize query passage pairs, and run inference, store results with the HuggingFace library.

What is meant by this is to store the results you achieved using the HuggingFace library, not to store them in the library.

**Comment by whoopi24** [comment]
Hi, 
I was also quite confused about the wording, maybe this answer helps: https://github.com/sebastian-hofstaetter/teaching/discussions/75

BR,
Marina
